kolibri/core/analytics/management/commands/benchmark.py

```diff
@@ -0,0 +1,137 @@
+
+import kolibri
+from kolibri.core.analytics.measurements import get_channels_usage_info
+from kolibri.core.analytics.measurements import get_db_info
+from kolibri.core.analytics.measurements import get_kolibri_process_cmd
+from kolibri.core.analytics.measurements import get_kolibri_use
+from kolibri.core.analytics.measurements import get_machine_info
+from kolibri.core.analytics.measurements import get_requests_info
+from kolibri.core.analytics.measurements import KolibriNotRunning
+from kolibri.core.analytics.pskolibri.common import LINUX
+from kolibri.core.analytics.pskolibri.common import WINDOWS
+from kolibri.utils.server import installation_type
+from kolibri.utils.system import get_free_space
+from kolibri.utils.time import local_now
+
+
+def format_line(parameter, value, indented=False):
+    if indented:
+        info = '  * {:30}'.format('{}:'.format(parameter))
+    else:
+        info = '* {:32}'.format('{}:'.format(parameter))
+    return '{info}{value}'.format(info=info, value=value)
+
+
+class Command(BaseCommand):
+    """
+    This command will output information about different parameters of the server running Kolibri
+    Output example:
+
+    Sessions
+    * Active sessions (guests incl): 10
+    * Active users in (10 min):      6
+    * Active users in (1 min):       3
+
+    CPU
+    * Total processes:               351
+    * Used CPU:                      33.6%
+    * Kolibri CPU usage:             22.3%
+
+    Memory
+    * Used memory:                   9.3 GB
+    * Total memory:                  16.0 GB
+    * Kolibri memory usage:          56.8 MB
+
+    Channels
+    * Total Channels:                2
+    * Khan Academy (English)
+      * Accesses:                    150
+      * Time spent:                  301.22 s
+    * African Storybook
+      * Accesses:                     3
+      * Time spent:                  18.00 s
+
+    Requests timing
+    * Homepage:                       0.03 s
+    * Recommended channels:           0.01 s
+    * Channels:                       0.02 s
+
+    Device info
+    * Version:                       (version)
+    * OS:                            (os)
+    * Installer:                     (installer)
+    * Database:                      (database_path)
+    * Device name:                   (device_name)
+    * Free disk space:               (content_storage_free_space)
+    * Server time:                   (server_time)
+    * Server timezone:               (server_timezone)
+    """
+    help = "Outputs performance info and statistics of usage for the running Kolibri instance in this server"
+
+    def handle(self, *args, **options):
+        if not LINUX and not WINDOWS:
+            print("This OS is not yet supported")
+            sys.exit(0)
+
+        try:
+            kolibri_cpu, kolibri_mem = get_kolibri_use()
+        except KolibriNotRunning:
+            sys.exit("Profile command executed while Kolibri server was not running")
```

Comment:
This should also be non-zero for the same reasons.. Does the string cause this? There is at least one more similar `sys.exit` call, so maybe just search for that string, I'll hold off with the duplicate code comments :)